compose/src/main/java/com/bottlerocketstudios/compose/map/YelpTestCard.kt

```diff
@@ -0,0 +1,11 @@
+package com.bottlerocketstudios.compose.map
+
+import com.bottlerocketstudios.mapsdemo.domain.models.Business
+import com.bottlerocketstudios.mapsdemo.domain.models.Coordinates
+
+val yelpTestCard = Business(
```

Comment:
@Josh-Gebbeken nit: This can be `internal`

```suggestion
internal val yelpTestCard = Business(
```